connector/microsoft/microsoft.go

```diff
@@ -51,6 +52,7 @@ type Config struct {
 	Groups               []string        `json:"groups"`
 	GroupNameFormat      GroupNameFormat `json:"groupNameFormat"`
 	UseGroupsAsWhitelist bool            `json:"useGroupsAsWhitelist"`
+	UpnToLowercase       bool            `json:"upnToLowercase"`
```

Comment:
I'm not intimately familiar with microsoft/AD, but I think it would make sense to use email as a phrase here. I guess the UPN is always an email address.